### PR TITLE
Variable font support for the Flutter Web CanvasKit renderer

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/flutter/issues/33709

### CODE REVIEW

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 1.
View them at https://flutter-engine-gold.skia.org/cl/github/35808

File: lib/web_ui/test/canvaskit/font_variation_golden_test.dart [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+      final CkParagraph paragraph = builder.build();
+      paragraph.layout(const ui.ParagraphConstraints(width: testWidth - 20));
+      canvas.drawParagraph(paragraph, const ui.Offset(10, 10));
+      CkPicture picture = recorder.endRecording();
+      await matchPictureGolden(
+        'font_variation.jpg',
+        picture,
+        region: ui.Rect.fromLTRB(0, 0, testWidth, paragraph.height + 20),
+        write: true
```
Comment (yjbanov) [inline lib/web_ui/test/canvaskit/font_variation_golden_test.dart:64]: Remove before submitting

**Review by yjbanov — APPROVED** [review]

File: lib/web_ui/test/canvaskit/font_variation_golden_test.dart (hunk 1 above)
Comment (jason-simmons) [reply lib/web_ui/test/canvaskit/font_variation_golden_test.dart:64]: done

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 1.
View them at https://flutter-engine-gold.skia.org/cl/github/35808

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 1.
View them at https://flutter-engine-gold.skia.org/cl/github/35808

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 1.
View them at https://flutter-engine-gold.skia.org/cl/github/35808

**Pull request merged**